img2dataset/logger.py

```diff
@@ -236,7 +236,12 @@ def run(self):
                 stats_files = fs.glob(output_path + "/*.json")
 
                 # filter out files that have an id smaller that are already done
-                stats_files = [f for f in stats_files if int(f.split("/")[-1].split("_")[0]) not in self.done_shards]
+                stats_files = [
+                    f
+                    for f in stats_files
+                    if not f.split("/")[-1].startswith("._")
```

Comment:
I think it would be much cleaner to do the reverse: check whether the file follows the expected filename structure. With a regex or explicit parsing